### ISSUE TITLE
Email validation does not set @details in the ActiveModel::Errors

### ISSUE THREAD

**Issue by IsaacOliveira** [lynndylanhurley/devise_token_auth#1351]
When the email validation fails the `DeviseTokenAuthEmailValidator` fills the @messages attribute in the error, but not the @details attribute introduced in Rails 5.

**Comment by MaicolBen** [comment]
Do you mean this [details](https://github.com/rails/rails/blob/b69ac962adeef897d4d1ba9aac40b9501ce25659/activemodel/lib/active_model/errors.rb#L342) ? Do you know how to do it? Because I'm not following what is the issue.

**Comment by IsaacOliveira** [comment]
Yes these details.
I am not using the gem anymore, but my guess is in the [DeviseTokenAuthEmailValidator](https://github.com/lynndylanhurley/devise_token_auth/blob/b409997d0fc03e05af61017a0dea290dcac71cb1/app/validators/devise_token_auth_email_validator.rb#L6)

We need to use this method to add the errors with messages and details:
https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add

Is someone reviewing PR's for the gem actively now?

**Comment by MaicolBen** [comment]
Yes, I'm reviewing the PRs, thank you in advance if you can do it!
